cmd/docs.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/TouchBistro/goutils/fatal"
+	"github.com/TouchBistro/tb/config"
+	"github.com/spf13/cobra"
+)
+
+var docsCmd = &cobra.Command{
+	Use:   "docs",
+	Args:  cobra.MinimumNArgs(1),
+	Short: "Opens link to API docs for a given service",
+	Run: func(cmd *cobra.Command, args []string) {
+		serviceName := args[0]
+		service, err := config.LoadedServices().Get(serviceName)
+		if err != nil {
+			fatal.ExitErrf(err, "%s is not a valid service.\nTry running `tb list` to see available services\n", serviceName)
+		}
+
+		if !service.HasGitRepo() {
+			fatal.Exitf("%s does not have a repo or is a third-party repo\n", serviceName)
+		}
+
+		docsURL := service.EnvVars["API_DOCS_URL"]
+		if docsURL == "" {
+			fatal.Exitf("API_DOCS_URL environment variable not found for service %s\n", serviceName)
+		}
```

Comment:
The problem with this is it requires env vars to be defined in the tb-registry. We pretty to have as many env vars in the `.env.example`s as possible and use the registry for stuff that is tb specific like URLs. You can scope the db cmd to see a hack that Omar did to get env vars from the container